analyze/workerd.ts

```diff
@@ -97,14 +123,22 @@ export async function isValidEmail(
   context: AnalyzeContext,
   candidate: string,
   options?: EmailValidationConfig,
-) {
+): Promise<EmailValidationResult> {
   const analyze = await init(context);
+  const optionsOrDefault = options || {
+    requireTopLevelDomain: true,
+    allowDomainLiteral: false,
+    blockedEmails: [],
+  };
 
   if (typeof analyze !== "undefined") {
-    return analyze.isValidEmail(candidate, options);
+    return analyze.isValidEmail(candidate, optionsOrDefault);
   } else {
     // TODO: Fallback to JS if we don't have WASM?
```

Comment:
```suggestion
```
We're not falling back because we're pretty certain we have Wasm in every environment we target.